openedx/edx-enterprise-data
# ENT-178: Fix django admin for edx-analytics-data-api

__Description:__
This PR movies model, admin and related definitions for user role functionality into a separate app so that proper database is used when operations are performed via these modes.

**TODO Before Merge:**
- [x] Upgrade enterprise data version.
- [x] Add Changelog entry for the changes

**TODO After Merge:**
- [x] Create a release tag for a PyPI release
- [x] Add the newly added app to `settings.INSTALLED_APPS` of `edx-analytics-data-api`

**Testing Instructions:**

- Install `edx-enterprise-data` in `edx-analytics-data-api` using this PR's branch
- Run `edx-analytics-data-api` server
- Visit Django admin portal and click on "Enterprise Data Role Assignment" (or visit http://localhost:8000/site/admin/enterprise_data_roles/enterprisedataroleassignment/)
- Make sure your able to create `Enterprise Data Role Assignment` records without any issue.
